The order of variable definitions and code can't be mixed with ISOC90

--- linux-sunxi-d47d367036be38c5180632ec8a3ad169a4593a88/drivers/net/wireless/rtxx7x/os/linux/usb_main_dev.c	2015-03-15 20:30:43.000000000 +0100
+++ linux-sunxi-d47d367036be38c5180632ec8a3ad169a4593a88/drivers/net/wireless/rtxx7x/os/linux/usb_main_dev.c	2016-12-11 23:17:00.710634654 +0100
@@ -444,11 +444,11 @@
 	struct net_device *net_dev;
 	VOID *pAd = usb_get_intfdata(intf);
 
-	late_resume_flag = TRUE;
-
 #ifdef USB_SUPPORT_SELECTIVE_SUSPEND
 	UCHAR Flag;
 #endif	
+
+	late_resume_flag = TRUE;
 #if defined(CONFIG_HAS_EARLYSUSPEND) || defined(CONFIG_ANDROID_POWER)
 	if ((RT_IS_EARLYSUSPEND_REGISTERED((PRTMP_ADAPTER)pAd)) && (late_resume_flag == TRUE)){
 		VIRTUAL_IF_DOWN((VOID *)pAd);
